build/system.gyp

```diff
@@ -35,6 +35,25 @@
        ],
       ],
     },
+    {
+      'target_name': 'libnotify',
+      'type': 'none',
+      'conditions': [
+        ['building_crosswalk_bin==1', {
+          'defines': ['USE_LIBNOTIFY=1'],
+          'cflags': [
+            '<!@(pkg-config --cflags libnotify)',
+            '<!@(pkg-config --cflags gdk-pixbuf-2.0)',
```

Comment:
Yeah, I think Peter was manually trying to work around the gdk-pixbuf errors the Linux builders were having. Don't, this is likely a packaging error in Ubuntu Precise's `libnotify-dev`. I've manually installed the gdk development packages in the bots so you shouldn't need this.